api/cpp/cmake/SlintMacro.cmake

```diff
@@ -50,5 +50,5 @@ function(SLINT_TARGET_SOURCES target)
 
         target_sources(${target} PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/${_SLINT_BASE_NAME}.h)
     endforeach()
-    target_include_directories(${target} PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+    target_include_directories(${target} PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
We need that because we need to make the header available to the main.cpp and to the test.

I wonder if that's right or if we should depends whether the Slint library is exposed private or public. Not sure there is a way to do that.

Reply:
This looks sensible to me. If somebody wants to "protect" this once more, they can use another level of indirection, i.e. use `slint_target_sources` on a `todo_ui` target and link that privately into the `todo_lib`, for example.